shared/js/ui/templates/top-blocked-truncated.es6.js

```diff
@@ -11,8 +11,8 @@ module.exports = function () {
       <ol class="default-list top-blocked__list top-blocked__list--truncated">
         ${listItems(this.model.companyListMap)}
       </ol>
-      <div class="top-blocked__see-all border--top">
-        <a href="#" class="link-secondary js-top-blocked-see-all">
+      <div class="top-blocked__see-all border--top js-top-blocked-see-all">
+        <a href="javascript:void(0)" class="link-secondary">
```

Comment:
Chrome complains about javascript:void(0): 
<img width="907" alt="screen shot 2018-01-17 at 1 00 45 pm" src="https://user-images.githubusercontent.com/3652195/35059683-fdfce8c6-fb89-11e7-993e-866208318877.png">

Reply:
Oh no -- CSPs. Ugh. We'll have to sync with Jason on this re: Safari.

Reply:
Lets just leave it as `#` for now, sorry for the confusion.